src/operators.jl

```diff
@@ -37,8 +36,7 @@ Types declared as safe can benefit from higher performance for operations on nul
 always computing the result even for null values, a branch is avoided, which helps
 vectorization.
 """
-null_safe_op(f::Any, ::Type) = false
-null_safe_op(f::Any, ::Type, ::Type) = false
+null_safe_op(f::Any, ::Type...) = false
```

Comment:
We still carry a copy of `null_safe_op` because it doesn't exist on 0.4, and even on 0.5 it only comes with definitions for `isless` and `isequal`. But indeed we should merge them at some point (I'll do that in another PR).